docs/admin-guide.md

```diff
@@ -0,0 +1,123 @@
+# Admin guide
+
+This guide is intended for Mattermost System Admins setting up the Bitbucket plugin and Mattermost users who want information about the plugin functionality. For more information about contributing to this plugin, visit the Development section. The Mattermost Bitbucket plugin uses a webhook to connect your Bitbucket account to Mattermost to listen for incoming Bitbucket events. Events notifications are via Direct Message in Mattermost. The Events don’t need separate configuration.
+
+- [Prerequsites](#prerequisites)
+- [Installation](#installation)
+  - [Marketplace](#marketplace)
+  - [Manual](#manual)
+  - [Cloud](#cloud)
+  - [Upgrade](#upgrade)
+- [Configuration](#configuration)
+- [Webhooks](#webhooks)
+- [Slash Commands](#slash-commands)
+- [Onboard Users](#onboard-users)
+- [FAQ](#faq)
+- [Get Help](#get-help)
+
+## Prerequisites
+
+This guide assumes:
+
+- You have a Bitbucket account.
+- You're a Mattermost System Admin.
+- You're running Mattermost Server v5.25 or higher.
+
+## Installation
+
+### Marketplace
+
+1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
+2. Search for "Bitbucket" or find the plugin from the list.
+3. Select **Install**.
+4. When the plugin has downloaded and been installed, select **Configure**.
```

Comment:
The plugin is not longer in the Marketplace. The manual way is the only way now.

Reply:
@hanzei I am a new contributor working here as part of Hacktoberfest so I wasn't aware of that. Should I delete the whole part?

Reply:
@kat-kan - Yes please. Delete the section about the Marketplace including the heading.